ncm-download/src/main/perl/download.pm

```diff
@@ -253,6 +254,7 @@ sub get_remote_timestamp
 sub retrieve
 {
     my ($self, $fn, %opts) = @_;
+    my $tmpfn = dirname($fn) . "/." . basename($fn) . ".part";
```

Comment:
I think (maybe wrong) I've seen this done in a lot of places, so didn't see much point, and it doesn't look like File::Spec is used currently. Happy to add it, though.